src/strongbox/ui/jfx.clj

```diff
@@ -767,7 +767,13 @@
   (let [idx-key #(select-keys % [:source :source-id])
         installed-addon-idx (mapv idx-key (fx/sub-val context get-in [:app-state :installed-addon-list]))
 
-        addon-list (fx/sub-val context get-in [:app-state :search-results])
+        search-state (fx/sub-val context get-in [:app-state :search])
```

Comment:
add comment about replication. if we find ourselves doing it again we can stick the logic somewhere nicer